content/design/pages/landing-page.md

```diff
@@ -0,0 +1,215 @@
+"""
+body = "Interested in becoming a champion of Spotlight PA at a higher level?"
+actionUrl = "https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&theme=leaders"
+actionText = "Become a leader →"
+url = "/support/leaders-in-action"
+
+[[blocks.entries]]
+image = "2022/06/01hb-4958-4rxz-4t0q.png"
+hed = "Option Two"
+body = "Want to support our investigative journalism about Penn State and our State College regional bureau?"
+actionUrl = "https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&theme=centrecounty&campaign=7015G0000013pZTQAY"
+actionText = "Support the Bureau →"
+url = "/support/state-college"
+
+[[blocks]]
+layout = "text-3"
+title = "Text Block Three"
+show_partners = 4
+copy = """
+
+  <p class="md:leading-relaxed">Spotlight PA’s unique model of public-service journalism relies on the generous support of individuals and institutions who understand the vital importance of Spotlight PA’s work to the future of Pennsylvania. The newsroom is supported by a dedicated group of local and national foundations, media partners, individual major donors (“Leaders in Action”), and thousands of readers from across Pennsylvania (“members”). Consistent with our mission, we disclose the source of every dollar we receive, and we do not accept anonymous gifts.</p>
+  
+  ## Learn more about who supports our work
+
+  [Institutional funders and members since launch →](/support/funders-and-members#major-donors-and-funders)
+
+  [Leaders in Action →](/support/leaders-in-action/#our-current-leaders)
+
+  [State College regional bureau supporters →](/support/state-college/#state-college-bureau-donors)
+"""
+
+[[blocks]]
+layout = "links"
+title = "Links Block"
+copy = "Spotlight PA was founded to address a crisis in journalism in Pennsylvania: the ongoing and rapid decline of corporate-owned legacy news outlets, which is greatly diminishing access to trusted, contextual, and factual reporting across the state. With support from foundations, individual members, and more, Spotlight PA is uncovering stories that would otherwise go untold and cutting through partisanship and misinformation to deliver trusted, reliable journalism at no cost to all in Pennsylvania. Since its inception in 2019, Spotlight PA has become a leading national model for independent, collaborative journalism that empowers residents and gets results. Its work has won numerous state and national awards and driven meaningful change and policy reforms in the state capital and beyond."
+
+
+[[blocks.entries]]
+link = "/donate/"
+name = "How to Give to Spotlight PA"
+
+[[blocks.entries]]
+link = "/about/partners/"
+name = "Partners"
+
+[[blocks.entries]]
+link = "/about/faq/"
+name = "FAQ"
+
+[[blocks.entries]]
+link = "/about/statecollege/"
+name = "State College Regional Bureau"
+
+[[blocks]]
+layout = "series"
+title = "Series"
+series = ["Standard Series", "The PSERS Scandal"]
+[blocks.cta]
+url = "/investigations/"
+copy = "See More About Our Impact"
```

Comment:
These blocks have yet another button style. We should add them to list of button styles and think about if we want to merge them away or keep them.